src/mobile/src/ui/components/CustomTextInput.js

```diff
@@ -439,41 +557,27 @@ class CustomTextInput extends Component {
                     ]}
                     testID={testID}
                 >
+                    {widgets.length === 2 && this.renderLeftHandWidget()}
                     <TextInput
-                        {...restProps}
                         ref={onRef}
-                        style={[styles.textInput, { color: theme.input.color }]}
+                        style={[
+                            styles.textInput,
+                            { color: theme.input.color, marginLeft: widgets.length === 2 ? 0 : width / 28 },
+                        ]}
                         onFocus={() => this.onFocus()}
                         onBlur={() => this.onBlur()}
+                        secureTextEntry={secureTextEntry && isSecretMasked}
```

Comment:
This will cause the mask button to not do anything if `secureTextEntry` is not `true` (which happens on several screens) even if `isSecretMasked` is `true`.

![image](https://user-images.githubusercontent.com/19519564/54642982-3a818180-4a6c-11e9-8bd4-47e8c2a22227.png)



This might be a better approach:

```suggestion
                        secureTextEntry={secureTextEntry ? true : isSecretMasked}
```

Reply:
@rajivshah3 Why not `secureTextEntry || isSecretMaskeed` ?

Reply:
I don't remember why I didn't suggest that 🤔 but yeah that should work